website/source/docs/provisioners/ansible.html.md

```diff
@@ -231,3 +241,17 @@ This template should build a Windows Server 2012 image on Google Cloud Platform:
   ]
 }
 ```
+
+### Too many SSH keys
+
+SSH servers only allow you to attempt to authenticate a certain number of times. All of your loaded keys will be tried before the dynamically generated key. If you have too many SSH keys loaded in your `ssh-agent`, the Ansible provisioner may fail authentication with a message similar to this:
```

Comment:
this is nuts! good find. For some reason I always assumed ssh-agent was smarter about this...